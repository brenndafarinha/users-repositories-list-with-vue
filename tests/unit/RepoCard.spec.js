import { shallowMount } from '@vue/test-utils';
import RepoCard from '../../src/components/RepoCard';

describe('RepoCard', () => {
    it("renders correctly", () => {
        const wrapper = shallowMount(RepoCard);
        expect(wrapper).toMatchSnapshot();
      });
});