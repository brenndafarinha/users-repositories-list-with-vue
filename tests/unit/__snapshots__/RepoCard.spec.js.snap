// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RepoCard renders correctly 1`] = `
<v-card hover="" width="400" min-height="150" class="d-flex flex-column justify-between mt-3 mx-2">
  <v-row class="d-flex align-start justify-between">
    <v-col cols="8">
      <v-card-title></v-card-title>
    </v-col>
    <v-col cols="4">
      <v-card-text class="d-flex align-center justify-end">
        <v-icon>mdi-star</v-icon> <span></span>
      </v-card-text>
    </v-col>
  </v-row>
  <v-card-subtitle></v-card-subtitle>
</v-card>
`;
